### PR TITLE
Goals are now reactive.

### PR DESCRIPTION
**Pull request by WouterKvG**

### CODE REVIEW

**Comment by WouterKvG** [comment]
@bartist @steffansluis PR!

File: app/routes/index/newgoals.js [hunk 1]
```diff
@@ -2,6 +2,15 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   model(){
-    return this.get('store').findAll('goal').then(function(goals) { return goals.sortBy('created_at').reverse(); });
+    return this.get('store').findAll('goal');
+  //   .then(function (data) {
```
Comment (bartist) [inline app/routes/index/newgoals.js]: Boooo, comments :-1:

**Comment by steffansluis** [comment]
Apart from the commented out code, I think it's good!

File: app/templates/index/mygoals.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{{mygoals/selection-goals model=model type="working"}}
+{{component 'bepstore-goals@goal/goal-mine' model=model type="working"}}
```
Comment (bartist) [inline app/templates/index/mygoals.hbs]: I think the "goal-mine" functionality is more appropriate in bepstore-ui than in ui-goals
Only bepstore-ui uses this functionality and it is not relevant for fbf to have it in the goal

Comment (WouterKvG) [inline app/templates/index/mygoals.hbs]: What is your opinion on this @steffansluis ?

Comment (steffansluis) [inline app/templates/index/mygoals.hbs]: I agree with @bartist, mostly because the sorting and filtering of goals is done based on things that are out of scope in bepstore-ui-goals, such as the current user.
